### ISSUE TITLE
Transform error when using TypeClone

### ISSUE THREAD

**Issue by wenerme** [sinclairzx81/typebox-workbench#8]
e.g. input

```ts
/**
 * @$id AA
 * @title AA
 */
export type AA = 'AA';

export interface X {
  /**
   * @title A
   */
  A: AA
}
```

after add imports, get require is undefined error because current project is esm.

**Comment by sinclairzx81** [comment]
@wenerme Hiya,

Yes, this looks like a bug, will look into it tomorrow. The repro code is very helpful! :)

**Comment by sinclairzx81** [comment]
Hiya,

This should be fixed on the latest publish

[Workbench Link Here](https://sinclairzx81.github.io/typebox-workbench/?share=LyoqCiAqIEAkaWQgQUEKICogQHRpdGxlIEFBCiAqLwpleHBvcnQgdHlwZSBBQSA9ICdBQSc7CgpleHBvcnQgaW50ZXJmYWNlIFggewogIC8qKgogICAqIEB0aXRsZSBBCiAgICovCiAgQTogQUEKfQ%3D%3D)

Also fixed on TypeBox-Codegen 0.9.1

Thanks for reporting!
S
